### PR TITLE
Update require to import statements

### PR DESCRIPTION
**Pull request by CallumGrindle**
Updates guides to use import statements instead of require to be up to date with ES6 standard JS

### CODE REVIEW

File: docs/guides/digital-assets/read-asset-data.md [hunk 1]
```diff
@@ -52,17 +52,17 @@ In the [**Read Profile Data** guide](../universal-profile/read-profile-data), we
...
+import Web3 from "web3";
+import { ERC725 } from "@erc725/erc725.js";
+import "isomorphic-fetch";
+import erc725schema from "@erc725/erc725.js/schemas/LSP3UniversalProfileMetadata.json";
 
 // Static variables
 const SAMPLE_PROFILE_ADDRESS = "0xa907c1904c22DFd37FF56c1f3c3d795682539196";
 const RPC_ENDPOINT = "https://rpc.l16.lukso.network";
-const IPFS_GATEWAY = "https://2eff.lukso.dev/ipfs/";
+const IPFS_GATEWAY = "https://cloudflare-ipfs.com/ipfs/";
```
Comment (fhildeb) [inline docs/guides/digital-assets/read-asset-data.md:63]: I would leave 2eff here, as the script does not work with cloudflare-ipfs.com (infinit load)

@Hugoo

File: docs/guides/digital-assets/read-asset-data.md [hunk 2]
```diff
@@ -859,24 +853,22 @@ Below is the complete code snippet of this guide, with all the steps compiled to
...
+import { ERC725Y_INTERFACE_IDS } from "@erc725/erc725.js/build/main/src/lib/constants";
 
 // Sample addresses
 const SAMPLE_PROFILE_ADDRESS = "0xa907c1904c22DFd37FF56c1f3c3d795682539196";
 const SAMPLE_ASSET_ADDRESS = "0x923F49Bac508E4Ec063ac097E00b4a3cAc68a353";
 
 // Network and storage
 const RPC_ENDPOINT = "https://rpc.l16.lukso.network";
-const IPFS_GATEWAY = "https://2eff.lukso.dev/ipfs/";
+const IPFS_GATEWAY = "https://cloudflare-ipfs.com/ipfs/";
```
Comment (fhildeb) [inline docs/guides/digital-assets/read-asset-data.md:871]: same as above.

File: docs/guides/universal-profile/read-profile-data.md [hunk 3]
```diff
@@ -64,17 +64,17 @@ We will use the convenient `fetchData()` function since we only need one command
...
+import Web3 from 'web3';
+import { ERC725 } from '@erc725/erc725.js';
+import 'isomorphic-fetch';
+import erc725schema from '@erc725/erc725.js/schemas/LSP3UniversalProfileMetadata.json';
 
 // Our static variables
 const SAMPLE_PROFILE_ADDRESS = '0xa907c1904c22DFd37FF56c1f3c3d795682539196';
 const RPC_ENDPOINT = 'https://rpc.l16.lukso.network';
-const IPFS_GATEWAY = 'https://2eff.lukso.dev/ipfs/';
+const IPFS_GATEWAY = 'https://cloudflare-ipfs.com/ipfs/';
```
Comment (fhildeb) [inline docs/guides/universal-profile/read-profile-data.md:75]: as above

File: docs/guides/universal-profile/read-profile-data.md [hunk 4]
```diff
@@ -219,17 +219,17 @@ Below is the complete code snippet of this guide, with all the steps compiled to
...
+import Web3 from 'web3';
+import { ERC725 } from '@erc725/erc725.js';
+import from 'isomorphic-fetch';
+import erc725schema from '@erc725/erc725.js/schemas/LSP3UniversalProfileMetadata.json';
 
 // Our static variables
 const SAMPLE_PROFILE_ADDRESS = '0x0C03fBa782b07bCf810DEb3b7f0595024A444F4e';
 const RPC_ENDPOINT = 'https://rpc.l16.lukso.network';
-const IPFS_GATEWAY = 'https://2eff.lukso.dev/ipfs/';
+const IPFS_GATEWAY = 'https://cloudflare-ipfs.com/ipfs/';
```
Comment (fhildeb) [inline docs/guides/universal-profile/read-profile-data.md:230]: as above

**Review by fhildeb — COMMENTED** [review]
Thanks for picking up this task and doing the upgrade. Only thing that needs to be discussed is the IPFS URL

**Comment by Hugoo** [comment]
Good point Felix - let's keep 2eff for now.

**Review by YamenMerhi — APPROVED** [review]

**Comment by CallumGrindle** [comment]
@fhildeb @Hugoo I already merged now after changing back to 2eff ipfs url. But realised we don't actually need this anyway as our libraries already use 2eff by default so there is no need to pass this in examples. I can make this change in my next PR

**Pull request merged**